nbformat/v4/nbformat.v4.schema.json

```diff
@@ -375,7 +375,7 @@
                 "patternProperties": {
                     "^application/([a-zA-Z0-9.]+\\+)?json$": {
                         "description": "Mimetypes with JSON output, can be an object",
-                        "type": "object"
+                        "type": ["object", "array"]
```

Comment:
Just comit the `type` property. See https://github.com/json-schema/json-schema/issues/172#issuecomment-114076650.